assets/src/dashboard/app/views/myStories/header/index.js

```diff
@@ -44,13 +45,19 @@ import {
   ViewPropTypes,
 } from '../../../../utils/useStoryView';
 import { useDashboardResultsLabel } from '../../../../utils';
-import {
-  BodyViewOptions,
-  HeaderToggleButtonContainer,
-  PageHeading,
-} from '../../shared';
+import { BodyViewOptions, PageHeading } from '../../shared';
 import { useConfig } from '../../../config';
+import { Pill } from '../../../../../design-system';
+
+const StyledPill = styled(Pill)`
+  margin: 0 2px;
+  white-space: nowrap;
 
+  & > span {
+    padding-left: 8px;
+    color: ${({ theme }) => theme.colors.fg.tertiary};
+  }
```

Comment:
Pill count for stories updates the design from using parenthesis to basing it on color.